src/app/admin/admin.module.ts

```diff
@@ -90,7 +90,6 @@ import { ApiKeyEditComponent } from './api-key/api-key-edit/api-key-edit.compone
     ApiKeyComponent,
     ApiKeyListComponent,
     ApiKeyTableComponent,
-    ApiKeyEditComponent,
```

Comment:
Is there a reason for this being deleted? I notice that the Organisation, Permission and User also are exporting the edit component.

Reply:
It's strictly not necessary. It's not being imported anywhere else, but I'll add it back for consistency